src/features/game/Session.tsx

```diff
@@ -1,18 +1,21 @@
-import React from "react";
+import React, {useRef} from "react";
 import ScrollContainer from "react-indiana-drag-scroll";
 
 import background from "assets/land/background.png";
 
 import { Game } from "./Game";
 import { GameProvider } from "./GameProvider";
 import { ToastProvider } from "./toast/ToastQueueProvider";
+import mapMovementSetup from "./lib/mapMovement";
 
 export const Session: React.FC = () => {
+  const container = useRef(null);
+  mapMovementSetup(container)
```

Comment:
If we only need to set this when the ref changes consider putting it inside of a useEffect.

E.g.

```
useEffect(() => {
  mapMovementSetup(container)
}, [container])
```